frontend/src/features/properties/list/buildings/Buildings.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { columns } from './columns';
+import { Table } from 'components/Table';
+import { IProperty } from './IProperty';
+
+/**
+ * Properties component properties
+ * @interface IProps
+ */
+export interface IProps {
+  data: IProperty[];
+  hideHeaders?: boolean;
+}
+
+/**
+ *
+ * @param {IProps} props Component properties.
```

Comment:
You may have forgotten to provide a body to your comment here.

Reply:
Will fix in next commit